ntirpc/rpc/xdr.h

```diff
@@ -257,7 +257,7 @@ typedef bool(*xdrproc_t) (XDR *, void *, u_int);
 /*
  * XXX can't actually prototype it, because some take three args!!!
  */
-typedef bool(*xdrproc_t) (XDR *, ...);
+typedef bool(*xdrproc_t) (XDR *, void *, unsigned int);
```

Comment:
u_int will always be unsigned int, as far as I know. You could just remove the `#ifdef` conditional and always use this typedef (i.e. spell out `unsigned int`)